Bottom: 04c3c88a00ab81eb4b92a4ae2f6067e9315d7198
Top:    2c7b1df59b1bd8d406963567864194d4559fe496
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-10-26 22:36:21 +0100

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
